exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/efgs/OperationDao.java

```diff
@@ -1,161 +1,127 @@
 package fi.thl.covid19.exposurenotification.efgs;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
 import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
-import org.springframework.jdbc.support.GeneratedKeyHolder;
-import org.springframework.jdbc.support.KeyHolder;
 import org.springframework.stereotype.Repository;
-import org.springframework.transaction.annotation.Transactional;
 
 import java.sql.Timestamp;
+import java.time.Duration;
 import java.time.Instant;
+import java.time.LocalDate;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
 import static java.util.Objects.requireNonNull;
 
 @Repository
 public class OperationDao {
+    public enum EfgsOperationState {STARTED, FINISHED, ERROR}
```

Comment:
This enums are used in a couple of places and their names clearly communicate enough context on their own (without the wrapping OperationDao...) -> I'd move them outside this class.

Reply:
Fixed imports. I'll put that moving on the todo list. They're kind of part of operation, I'm not much of fan of globals. Let's see where this goes.